### PR TITLE
create model card for astroGPT

### PR DESCRIPTION
**Pull request by stevhliu**

### CODE REVIEW

File: model_cards/stevhliu/astroGPT/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+language: "en"
+thumbnail: "https://github.com/stevhliu/satsuma/blob/master/images/astroGPT-thumbnail.png"
```
Comment (julien-c) [inline model_cards/stevhliu/astroGPT/README.md:3]:
```suggestion
thumbnail: "https://raw.githubusercontent.com/stevhliu/satsuma/master/images/astroGPT-thumbnail.png"
```

File: model_cards/stevhliu/astroGPT/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Data
+
+The data was scraped from [Horoscope.com](https://www.horoscope.com/us/index.aspx) and trained on 4.7MB of text. The text was collected from four categories (daily, love, wellness, career) and span from 09/01/19 to 08/01/2020. The archives only store horoscopes dating a year back from the current date.
+
+## Training and results
+
+The text was tokenized using the fast GPT-2 BPE [tokenizer](https://huggingface.co/transformers/model_doc/gpt2.html#gpt2tokenizerfast). It has a vocabulary size of 50,257 and sequence length of 1024 tokens. The model was trained with on one of Google Colaboratory's GPU's for approximately 2.5 hrs with [fastai's](https://docs.fast.ai/) learning rate finder, discriminative learning rates and 1cycle policy. See table below for a quick summary of the training procedure and results.
```
Comment (julien-c) [inline model_cards/stevhliu/astroGPT/README.md:47]: ^^ cc @sgugger ;)

**Comment by julien-c** [comment]
really cool model, thanks for sharing:

Screenshot 2020-09-05 at 18 49 27

**Comment by stevhliu** [comment]
ヘ( ^o^)ノ＼(^_^ ) thanks @julien-c

**Pull request merged**